modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/fragments/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "extra-liferay-fragments",
```

Comment:
Probably, this is a bad name.

Following the existing pattern, would be:

`@liferay/osb-site-initializer-customer-portal-fragments`